Framework/Algorithms/src/FitPeaks.cpp

```diff
@@ -1281,6 +1292,13 @@ void FitPeaks::fitSpectrumPeaks(size_t wi, const std::vector<double> &expected_p
                                bkgdfunction, peak_pre_check_result);
       if (peak_pre_check_result->isIndividualPeakRejected())
         fit_result->setBadRecord(peak_index, -1.);
+
+      auto signalToSigma = calculateSignalToSigmaRatio(wi, peak_window_i, peakfunction, bkgdfunction);
```

Comment:
I think this should only be calculated if `m_minSignalToSigmaRatio > 0` - so we don't affect performance of existing scripts. What do you think?

Reply:
Yes, right, good catch